net/message/block.go

```diff
@@ -25,17 +25,15 @@ type block struct {
 }
 
 func (msg block) Handle(node Noder) error {
-	log.Debug()
-
 	log.Debug("RX block message")
-
-	err := ledger.DefaultLedger.Blockchain.AddBlock(&msg.blk)
-	if err != nil {
-		log.Error("Add block error after Received")
-		return errors.New("Add block error after reveived\n")
+	if !ledger.DefaultLedger.ContainBlock(msg.blk.Hash()) {
```

Comment:
the logic like below will be better:
if (something already existed) {
     return 
}

then 
do the normal process